android/sample/src/main/java/com/boswelja/ephemeris/sample/compose/ComposeCalendarFragment.kt

```diff
@@ -42,15 +50,33 @@ class ComposeCalendarFragment : Fragment() {
 }
 
 @Composable
-fun CalendarScreen() {
+fun CalendarScreen(
+    headerDateFormatter: DateTimeFormatter = DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM)
+) {
     val calendarState = rememberCalendarState {
         CalendarMonthPageSource(
             DayOfWeek.SUNDAY
         )
     }
+
+    var headerText by remember { mutableStateOf("") }
+
+    LaunchedEffect(calendarState) {
+        calendarState.displayedDateRange.collectLatest {
+            headerText = "${it.start.toJavaLocalDate().format(headerDateFormatter)} - ${it.endInclusive.toJavaLocalDate().format(headerDateFormatter)}"
```

Comment:
>long line of code
tempted to say extension function here instead? Or at least some line breaks 😅

Reply:
An extension makes sense, we can share it with XML views that way

Reply:
I've created an issue #52 to update sample code